src/store/entities/selectors.ts

```diff
@@ -0,0 +1,71 @@
+import set from 'lodash.set';
+import get from 'lodash.get';
+
+import decodeQueryCacheKey from '../../utils/decodeQueryCacheKey';
+
+import { State } from '../reducer';
+import { State as EntitiesState } from '../entities/reducer';
+import { Entity } from '../../typings/API';
+
+/**
+ * @TODO
+ * Helper function to determine how to convert an api entity reference; f.e.
+ * { type: 'contact', id: 'test123' }
+ * into the domain type "contacts", should this be revised in the api or should we
+ * change our store to store directly by type, or create a mapping?
+ */
+export const convertAPIEntityTypeToDomain = (type: string) => {
+  return `${type}s`;
```

Comment:
Not sure if I'm following here, but what about `company`?

Reply:
Changed to a mapping in a later commit 🙈

Reply:
I feel like we could also solve this somewhere else though, I see that the `api-gateway` also has the exact same mapping but reversed and imo it just makes it less flexible

Reply:
Ah yeah checked it commit by commit 👍